site/content/how-to/monitoring/prometheus.md

```diff
@@ -102,11 +102,11 @@ You can configure monitoring metrics for NGINX Gateway Fabric using Helm or Mani
 
 ### Using Helm
 
-If you're setting up NGINX Gateway Fabric with Helm, you can adjust the `metrics.*` parameters to fit your needs. For detailed options and instructions, see the [Helm README](/charts/nginx-gateway-fabric/README.md).
+If you're setting up NGINX Gateway Fabric with Helm, you can adjust the `metrics.*` parameters to fit your needs. For detailed options and instructions, see the [Helm README](https://github.com/nginxinc/nginx-gateway-fabric/blob/v1.3.0/charts/nginx-gateway-fabric/README.md).
```

Comment:
Should these point to main in the main branch so they can be updated to the release version when we run the release PR?

Reply:
I think we only update from one tag to another?

Reply:
Ok as long as they're updated properly, wasn't sure if we only update `main` references or not.